draft-ietf-quic-recovery.md

```diff
@@ -216,23 +215,22 @@ implemented in QUIC.
 
 An unacknowledged packet is marked as lost when an acknowledgment is received
 for a packet that was sent a threshold number of packets (kReorderingThreshold)
-after the unacknowledged packet. Receipt of the ack indicates that a later
-packet was received, while kReorderingThreshold provides some tolerance for
-reordering of packets in the network.
-
-The RECOMMENDED initial value for kReorderingThreshold is 3.
-
-We derive this recommendation from TCP loss recovery {{?RFC5681}}
-{{?RFC6675}}. It is possible for networks to exhibit higher degrees of
-reordering, causing a sender to detect spurious losses. Detecting spurious
-losses leads to unnecessary retransmissions and may result in degraded
+and/or a threshold amount of time after the unacknowledged packet. Receipt of the
+ack indicates that a later packet was received, the reordering threshold
+provides some tolerance for reordering of packets in the network.
+
+The RECOMMENDED initial value for kReorderingThreshold is 3, based on 
+TCP loss recovery {{?RFC5681}} {{?RFC6675}}. Some networks may exhibit higher
+degrees of reordering, causing a sender to detect spurious losses. Detecting
+spurious losses leads to unnecessary retransmissions and may result in degraded
```

Comment:
Perhaps it's simply a transport term of art, but talking about "detecting spurious losses" has the wrong connotation to me.  When we say that you've detected a spurious retransmission, it means you realized that retransmission was actually unnecessary; detecting spurious losses kind of feels like realizing a packet you thought was lost actually wasn't.

I know you're actually trying to talk about the initial incorrect conclusion, not the later realization it was incorrect.  Maybe it would be clearer to say that highly reordering networks "caus[e] a sender to incorrectly mark packets as lost.  Incorrect loss detection leads to...."?

Reply:
Agreed, its either confusing or wrong, depending on your perspective.

How about spuriously detecting losses?

Reply:
somehow reminds me of https://en.wikiquote.org/wiki/Colossal_Cave_Adventure